application/common/mail/reset/self.php

```diff
@@ -0,0 +1,15 @@
+Hi there,
+
+Someone recently requested a password change for your <?php echo \Yii::$app->params['appName']; ?>
+account. If this was you, you can use the reset code below to set a new password.
+
+Reset Code: <?php echo \yii\helpers\Html::encode($resetCode); ?> 
+
+If you don't want to change your password or didn't request this, just 
+ignore and delete this message. 
+
+To keep your account secure, please don't forward this email to anyone. 
+See our Help Center for <a href="<?php echo \Yii::$app->params['helpCenterUrl']; ?>">more security tips</a>.
```

Comment:
I was just planning that there would be one as part of our default config. Technically it could not exist and then cause an error here, so perhaps I should, but it might be better to get an error so it can be fixed rather than silently working.